JULIERAJ/KIDS-FIRST-Front
# create co-parent chatting screen

## One Line Description
Implemented a co-parent chatting screen

## Test Steps
1. Open the application.
2. Navigate to 'dashboard/chatscreen' route.
3. Verify the styles and alignments for each section to ensure they match the Figma design specifications.

## UI/UX Outcome
- ![image](https://github.com/user-attachments/assets/db3e8db7-5313-4bba-a585-f30fcef14e07)
[Insert relevant screenshots or videos here]

## Checklist
